float64 initial_value
float64 min
float64 max
float64 uncertainty
float64 scaling
